tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_realtime_unsupported.py

```diff
@@ -0,0 +1,76 @@
+# cat test_realtime_unsupported.py
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+
+import pytest
+from wazuh_testing import global_parameters
+from wazuh_testing.fim import generate_params, regular_file_cud, LOG_FILE_PATH, callback_num_inotify_watches, \
+                              detect_initial_scan, CHECK_ALL, REQUIRED_ATTRIBUTES
+from wazuh_testing.tools import PREFIX
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+
+# Marks
+
+pytestmark = [pytest.mark.darwin, pytest.mark.sunos5, pytest.mark.tier(level=0)]
+
+# variables
+
+
+test_directories = [os.path.join(PREFIX, 'dir')]
+
+directory_str = str(test_directories[0])
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_conf_check_realtime.yaml')
+testdir = test_directories
+test_file = "testfile.txt"
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+# configurations
+
+
+conf_params = {'TEST_DIRECTORIES': directory_str, 'MODULE_NAME': __name__}
+parameters, metadata = generate_params(extra_params=conf_params, modes=['scheduled'])
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+
+# fixtures
+
+
+@pytest.fixture(scope='function', params=configurations)
+def check_realtime_mode_failure():
```

Comment:
Instead of waiting for a minute before starting the test, we could change this fixture to look for the log message stating: `Ignoring flag for real time monitoring on directory: ...`

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1474/commits/a46ae8a444c0c0a7f68436a4ccee7af9780437c6)